Source/WebCore/platform/graphics/gbm/DMABufObject.h

```diff
@@ -40,20 +40,6 @@
 namespace WebCore {
 
 struct DMABufObject {
-    WTF_MAKE_NONCOPYABLE(DMABufObject);
```

Comment:
Why do we need to remove `WTF_MAKE_NONCOPYABLE()` here? As far as I tried, the autogenerated serialization code can deal with non-copyable objects.

Reply:
Also the generated code uses move semantics, so if this is getting copied somewhere, that should be fixed instead.

Reply:
Adding any non-default implicit move constructor disables the implicit move assignment operator, and I suspect that's why Pablo removed `WTF_MAKE_NONCOPYABLE()`; the correct solution is to bring back the default move assignment operator with:

```cpp
DMABufObject& operator=(DMABufObject&&) = default;
```

I have a version of this patch that addresses the review issues and keeps the `DMABufObject` class as non-copyable, but I will have to submit it as a different PR.